features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Throttling/Subscription/AddEdit.jsx

```diff
@@ -239,8 +239,10 @@ function AddEdit(props) {
                     customAttributes: setCustomAttributes(result.body.customAttributes),
                     stopOnQuotaReach: result.body.stopOnQuotaReach,
                     permissions: {
-                        roles: 'Internal/everyone',
-                        permissionStatus: 'ALLOW',
+                        permissionStatus: (result.body.permissions === null)
+                            ? 'ALLOW' : result.body.permissions.permissionType,
+                        roles: (result.body.permissions === null)
+                            ? 'Internal/everyone' : result.body.permissions.roles.join(','),
```

Comment:
roles: (result.body.permissions === null)
                            ? 'Internal/everyone' : result.body.permissions.roles && result.body.permissions.roles.join(','),

@menakaj please validate the above fix

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/9504/commits/e8a9cf0eb1ec6e36c8c12e4d9ef3f76835392102